app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewActivity.kt

```diff
@@ -60,11 +68,28 @@ class PledgedProjectsOverviewActivity : AppCompatActivity() {
                         modifier = Modifier,
                         onBackPressed = { onBackPressedDispatcher.onBackPressed() },
                         lazyColumnListState = lazyListState,
+                        errorSnackBarHostState = snackbarHostState,
                         ppoCards = ppoCardPagingSource,
-                        totalAlerts = totalAlerts.value
+                        totalAlerts = totalAlerts.value,
+                        onSendMessageClick = { projectName -> viewModel.onMessageCreatorClicked(projectName) }
                     )
                 }
 
+                LaunchedEffect(Unit) {
+                    viewModel.projectFlow
+                        .collect {
+                            startCreatorMessageActivity(it, previousScreen = MessagePreviousScreenType.PLEDGED_PROJECTS_OVERVIEW)
+                        }
+                }
+
+                val coroutineScope = rememberCoroutineScope()
+
+                viewModel.provideSnackbarMessage {
+                    coroutineScope.launch {
```

Comment:
I recommend you use `lifecycleScope` here as it is more aware of the activity lifecycle and will close the job if the activity/fragment dies:
https://developer.android.com/topic/libraries/architecture/coroutines#lifecyclescope

Reply:
Ah okay, i did see that as another option, though I wasn't sure if it was overkill. I'll definitely update!!